src/components/Sidebar/style.scss

```diff
@@ -1,84 +1,129 @@
-          border-radius: 2px;
-          margin-top: 20px;
-          margin-bottom: 50px;
-        }
-      }
-    }
-  }
-
-  .sidebar-icon {
-    width: 25px;
-    height: 20px;
-    display: inline-block;
-  }
-
-  .my-resources-icon {
-    background: url(../../assets/images/my-resource-icon.png) no-repeat 0 7px;
-  }
-
-  .teams-icon {
-    background: url(../../assets/images/teams-icon.png) no-repeat 0 7px;
-  }
-
-  .favorites-icon {
-    background: url(../../assets/images/favorites-icon.png) no-repeat 0 7px;
-  }
-
-  .recent-resource-icon {
-    background: url(../../assets/images/recent-resource-icon.png) no-repeat 0 7px;
-  }
+	background: #fff;
+	min-height: 900px;
+	padding-top: 30px;
+	padding-right: 0px;
+
+	& > ul {
+		list-style: none;
+		padding: 0px 30px;
+
+		& > li {
+			border-top: 1px solid #e8e8e8;
+
+			a {
+				color: #607a9b;
+			}
+
+			& > a {
+				font-size: 20px;
+				display: block;
+				padding-top: 20px;
+				padding-bottom: 20px;
+
+				&:hover {
+					text-decoration: none;
+				}
+			}
+
+			&:last-child {
+				border-bottom: none;
+			}
+
+			.sublist {
+				list-style: none;
+				padding-left: 25px;
+
+				a {
+					display: block;
+					padding-top: 5px;
+					padding-bottom: 5px;
+					font-size: 14px;
+				}
+			}
+
+			.add-member {
+				padding-left: 25px;
+
+				button {
+					color: #607a9b;
+					border: 1px solid #bfbfbf;
+					cursor: pointer;
+					padding: 5px 10px 5px 35px;
+					background: #fafafa url(../../assets/images/add-member-icon.png) no-repeat
+						10px 9px;
+					border-radius: 2px;
+					margin-top: 20px;
+					margin-bottom: 50px;
+				}
+			}
+		}
+	}
+
+	.sidebar-icon {
+		width: 25px;
+		height: 20px;
+		display: inline-block;
+	}
+
+	.my-resources-icon {
+		background: url(../../assets/images/my-resource-icon.png) no-repeat 0 7px;
+	}
+
+	.teams-icon {
+		background: url(../../assets/images/teams-icon.png) no-repeat 0 7px;
+	}
+
+	.favorites-icon {
+		background: url(../../assets/images/favorites-icon.png) no-repeat 0 7px;
+	}
+
+	.recent-resource-icon {
+		background: url(../../assets/images/recent-resource-icon.png) no-repeat 0 7px;
+	}
+}
 
+.sidebarall {
+	.expand {
+		text-align: left;
+		text-decoration: underline;
+		font: normal normal normal 13px/22px Open Sans;
+		letter-spacing: 0px;
+		color: #0f4392;
+		opacity: 1;
+		border-top: 1px solid #e8e8e8;
+		width: 100%;
+		display: block;
+		svg {
+			margin-left: 10px;
+		}
+	}
+	.menu-title {
+		text-align: left;
+		font: normal normal bold 16px/25px Open Sans;
+		letter-spacing: 0px;
+		color: #0f4392;
+		text-transform: uppercase;
+		opacity: 1;
+
+		font-weight: 600;
+		padding: 12px 20px;
+		border-top: 1px solid;
+		border-top: 1px solid #e8e8e8;
+
+		&:hover {
+			cursor: pointer;
+			text-decoration: none !important;
+		}
+	}
+
+	a {
+		text-align: left;
+		font: normal normal 600 14px/25px Open Sans;
+		letter-spacing: 0px;
+		color: #464646;
+		opacity: 1;
+		padding: 10px 0px;
+		font-weight: 400;
+	}
```

Comment:
Is this just indentation changes?

Reply:
Do not add useless indentation changes please
It makes confusing while reviewing

Reply:
since I have removed prettier so it was little time taking to indent. so i used an online formatter for this piece of code.

Reply:
Okay, skipping for now, but please use 2 spaces indent when you use online tool
I think there will be option for that